vars/runSctTest.groovy

```diff
@@ -35,7 +35,7 @@ def call(Map params, String region){
 
     if [[ -n "${params.scylla_ami_id ? params.scylla_ami_id : ''}" ]] ; then
         export SCT_AMI_ID_DB_SCYLLA="${params.scylla_ami_id}"
-    elif [[ ! -z "${params.gce_image_db}" ]] ; then
+    elif [[ -n "${params.gce_image_db ? params.gce_image_db : ''}" ]] ; then
```

Comment:
first glimpse, and I was thing is that something new I don't know in bash.

mixing the logic of groovy with bash one, is going to be a bit confusing most people. 
we might want to put that logic outside the bash part, for clarity.

Reply:
I don't mind having this logic in a separate script which can be just called here. But, I think, it is out of scope for this small "fix" of it's logic.

Reply:
agree with Valerii, this is just a fix